monitor/cli.py

```diff
@@ -1,19 +1,24 @@
-import click
 import os
 from os.path import expanduser
 
-from utils.package import get_package_version
+import click
+
+from config.config import Config
+from monitor.data_monitoring import DataMonitoring
 from tracking.anonymous_tracking import AnonymousTracking, track_cli_start, track_cli_exception, track_cli_end
 from utils.cli_utils import RequiredIf
+from utils.log import get_logger
 from utils.ordered_yaml import OrderedYaml
-from config.config import Config
-from monitor.data_monitoring import DataMonitoring
+from utils.package import get_package_version
 
 yaml = OrderedYaml()
 
+logger = get_logger(__name__)
 
-def get_cli_properties() -> dict:
+ERR_MISSING_SLACK_CONN = 'Either a Slack token or webhook is required to send the report.'
```

Comment:
cosmetics - since you use this message in the 'monitor' command maybe say something - 
'Either a Slack token or webhook is required'